folding-schemes/src/folding/circuits/mod.rs

```diff
@@ -8,6 +41,9 @@ pub mod nonnative;
 pub mod sum_check;
 pub mod utils;
 
+// TODO (autoparallel): nit -- I think the short variable names `CF1` and `CF2` could be made more clear by something
+// like `ConstraintFieldPrimary` and `ConstraintFieldSecondary`, but this isn't a hill I'll die on
```

Comment:
Agree on that their names are not super clear at first sight since it's an acronym name, the fear is that those are used a lot along the code and having a short name helps with code readability :S
(used a lot and in combined cases, like `CurveVar<C1, CF2<C1>>`)
So far we've tried to have an explanation of each of them in the comment of their definition, so that it leaves not doubt about what are each of them.